src/Renderer/Camera.js

```diff
@@ -124,11 +124,13 @@ Camera.prototype.computeNodeSSE = function(node) {
     var boundingSphere = node.geometry.boundingSphere;
     var distance = Math.max(0.0, (this.camera3D.position.distanceTo(node.centerSphere) - boundingSphere.radius));
     // Added small oblique weight (distance is not enough, tile orientation is needed)
+    /*
```

Comment:
You're switching back to the old SSE?
Documenting why (in commit message) would be interesting - and maybe change the commit's title btw: you're doing more than juste `AddNew Api Functions`.